### PR TITLE
Did challenge 1

### PR DESCRIPTION
**Pull request by djfigueroa**
Did challenge 1
@RIckyC

### CODE REVIEW

File: Week 7/Practice1.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    </head>
+    <body>
+        <p>My name is <span id="myName"></span></p>
+        <script>
+            const theName = "Shrek";
+            $("span#myName").append(`${theName}`)
```
Comment (rickyc) [inline Week 7/Practice1.html:10]: 👍 Looks good! You can even simplify it more by doing the following.

```js
$("span#myName").append("Shrek");
```

File: Week 7/Practice10.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        </head>
+        <body>
+            <h2>List of Characters</h2>
+            <ul id="names">
+            </ul>
+            <p id="Unorder"></p>
+            <button type="button" onclick="nameList.forEach(Alphabetical)">Alphabetical order</button>
+            <script>
+                Alpha = document.getElementById("Unorder");
```
Comment (rickyc) [inline Week 7/Practice10.html:13]: Make sure variables are lowercase and use a `const`.

```js
const alpha = $("#Unorder");
```

Also make sure `id`s are in lowercase.

File: Week 7/Practice2.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```
Comment (rickyc) [inline Week 7/Practice2.html:4]: 👍 Looks good.

File: Week 7/Practice2.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <body>
+        <p>My name is <span id="myName"></span></p>
+        <script>
+            const theNumber = parseInt(Math.random() * 2);
+            if (theNumber == 0) {
+            	$("span#myName").html("Shrek")
+            }
+            else {
+            	$("span#myName").html("Princess Fiona")
+            }
```
Comment (rickyc) [inline Week 7/Practice2.html:15]: You can even do some more magic by writing it as follows.

```js
const name = theNumber == 0 ? "Shrek" : "Princess Fiona";
```

File: Week 7/Practice3.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+<html>
+    <head>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    </head>
+    <body>
+        <p>My name is <span id="myName"></span></p>
+        <button type="button">Click Me!</button>
+        <script>
+            const theName = "Shrek";
+            $("button").click(()=>{
```
Comment (rickyc) [inline Week 7/Practice3.html:11]: 👍 Looks good! You can simplify this even further like this

```js
$("button").click(() => $("span#myName").html("Shrek"));

File: Week 7/Practice4.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+<html>
+    <head>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    </head>
+    <body>
+        <input type="text">
+        <button type="button">Click Me!</button>
+        <p>My name is <span id="myName"></span></p>
+        <script>
+            $("button").click(() => {
```
Comment (rickyc) [inline Week 7/Practice4.html:11]: 👍

File: Week 7/Practice5.html [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+        <body>
+            <input type="radio" name="radioPlace" id="NYC" value="nyc">
+            <label for="radioButton">New York</label>
+            <input type="radio" name="radioPlace" id="Bostonian" value="bost">
+            <label for="radioButton">Boston</label>
+            <p><span id="myName"></span></p>
+            <button type="button">Click Me!</button>
+            <script>
+                $("button").click(() => {
```
Comment (rickyc) [inline Week 7/Practice5.html:15]: ❌ See what `$("input:checked").val()` returns using the Chrome console.

File: Week 7/Practice6.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+        <head>
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        </head>
+        <body>
+            <p>My name is <span id="myName"></span></p>
+            <button>Click Me!</button>
+            <script>
+                const nameList = ["Shrek","Donkey","Princess Fiona"];
+                $("button").click(()=>{
+                var random = nameList[Math.floor(Math.random() * 3)];
```
Comment (rickyc) [inline Week 7/Practice6.html:12]: This should be a `const`. Watch for indentation and use `newList.length` to get the length of the array. Don't use the number 3 as the array size can change.

File: Week 7/Practice6.html [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+                $("span").html(`${random}`)
```
Comment (rickyc) [inline Week 7/Practice6.html:13]: This can be rewritten as follows.

```js
$("span").html(random);
```

File: Week 7/Practice9.html [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+        <body>
+            <img src="https://upload.wikimedia.org/wikipedia/commons/0/08/EmpireStateNewYokCity.jpg" width="350" id="NYC"/>
+            <img src="https://media-cdn.tripadvisor.com/media/photo-s/03/9b/2f/47/boston.jpg" width="350" id="Boston"/>
+            <p id="text"></p>
+            <h1>NYC Counter</h1>
+            <p id="Nyc"></p>
+            <h1>Boston Counter</h1>
+            <p id="BOSTON"></p>
+            <script>
+                let counterNYC = 0;
```
Comment (rickyc) [inline Week 7/Practice9.html:15]: 👍

File: Week 7/Practice7&8.html [hunk 11]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+    <html>
+        <head>
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        </head>
+    
+        <body>
+
```
Comment (rickyc) [inline Week 7/Practice7&8.html:8]: 👍

**Review by rickyc — APPROVED** [review]
👍 LGTM (Looks good to me). Merge it!
